lib/fastlane/plugin/fueled/actions/formatting_checks_flutter.rb

```diff
@@ -7,7 +7,7 @@ class FormattingChecksFlutterAction < Action
       def self.run(params)
         UI.message("Checking formatting")
         sh("flutter pub get")
-        sh("dart format -o none --set-exit-if-changed .")
+        sh("dart format $(find lib test integration_test -name \"*.dart\" -not \( -name \"*.*freezed.dart\" -o -name \"*.g.dart\" -o -name \"*.gen.dart\" -o -name \"*.mocks.dart\" \) ) -o none --set-exit-if-changed")
```

Comment:
This change was needed as dart format does not exclude the files specified in analysis options. [Reference](https://github.com/dart-lang/dart_style/issues/864#issuecomment-1233176900)

Reply:
can we pull the internal command into a local variable for readability.

Reply:
done!